meta-iot2000-bsp/scripts/lib/wic/canned-wks/wic-image.iot2000.wks

```diff
@@ -1,5 +1,9 @@
+# short-description: Create an iot2000 example disk image
+# long-description: Creates a partitioned EFI disk image for the iot2000,
+# using efibootguard, that the user can directly dd to boot media.
+
 include common.iot2000.wks.header.inc
 
-part / --source rootfs --ondisk mmcblk0 --fstype=ext3 --label platform --align 1024
```

Comment:
Hm, is there a reason this was `ext3`? If not, OK, otherwise I'd like to hear the reason to change this.

Reply:
Yes and I like to hear the reason why it should stay ext3 IF it must stay ext3 first.

Reply:
It mustn't stay. I just like to hear the reasoning why it was `ext3` in the first place and then why you changed it to be `ext4`. Could be there is one, could be there's none and in the latter case it's fine - if you give a reason, that is :)

Reply:
My reason was just to update it so that it's newer. For example ext4 has journal checksum or fast fsck, which ext3 has not.

Reply:
I'm fine with going for ext4, in a separate commit of course.